docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -2236,6 +2236,20 @@ How to **fix** it:
 
 - Increase the allowed limit by using the `-distributor.max-recv-msg-size` option.
 
+### err-mimir-distributor-max-otel-decompressed-write-message-size
+
+This error occurs when a distributor rejects an OTEL write request because its message size is larger than the allowed limit after decompression.
+
+How it **works**:
+
+- The distributor implements an upper limit on the message size of incoming otel write requests after decompression regardless compression type. See more otlp collector [compression details](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/README.md#client-configuration).
+- To configure the limit, set the `-distributor.max_otel_decompressed_recv_msg_size`
+
+How to **fix** it:
+
+- Increase the allowed limit by using the `-distributor.max_otel_decompressed_recv_msg_size` option.
```

Comment:
Same mistyping as above:

Reply:
flag updated in commit https://github.com/grafana/mimir/commit/1ba1bcb4854d016dfada97077fc1696ea23c84e3